Tests/GRPCTests/GRPCReflectionServiceTests/Utils.swift

```diff
@@ -116,3 +129,9 @@ extension Sequence where Element == Google_Protobuf_EnumDescriptorProto {
     self.map { $0.name }
   }
 }
+
+extension Sequence where Element == Google_Protobuf_FieldDescriptorProto {
+  var names: [String] {
+    self.map { $0.name }
+  }
+}
```

Comment:
This extension is so simple that I don't think it's worth defining, especially as it's only used in one place.